examples/bite_sized/conditional_branching.py

```diff
@@ -1,13 +1,44 @@
 import csv
+import os
 import xml.etree.ElementTree as ET
 from unittest.mock import MagicMock
 
+import pandas as pd
 import requests
-from dagster import Out, Output, job, op, resource
+from dagster import (
+    IOManager,
+    InputContext,
+    Out,
+    Output,
+    OutputContext,
+    io_manager,
+    job,
+    op,
+    resource,
+)
 
 ARTICLES_LINK = "https://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml"
 
 
+class DataframeToCSVIOManager(IOManager):
+    def __init__(self, base_dir):
```

Comment:
Worth including a type annotation for base_dir as well as return type annotations for the other functions.